pkg/apis/scope/v0alpha1/types.go

```diff
@@ -60,9 +60,16 @@ type ScopeDashboardBinding struct {
 }
 
 type ScopeDashboardBindingSpec struct {
-	Dashboard      string `json:"dashboard"`
+	Dashboard string `json:"dashboard"`
+
+	// DashboardTitle should be populated and update from the dashboard
 	DashboardTitle string `json:"dashboardTitle"`
 
+	// Groups is used for the grouping of dashboards that are suggested based
+	// on a scope. The source of truth for this information has not been
+	// determined yet.
+	Groups []string `json:"groups,omitempty"`
```

Comment:
will `DashboardTitle` and `Groups` be updated by something watching the dashboard for updates?

Reply:
> will `DashboardTitle` and `Groups` be updated by something watching the dashboard for updates

That is the plan for `DashboardTitle`, where `Groups` will come from is still unknown.

Reply:
> That is the plan for DashboardTitle

i'd suggest moving that to [status](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status). here's an [example](https://github.com/grafana/grafana/blob/eecd56659c4f3512b27cb6d741540fe67ae1b069/pkg/aggregator/apis/aggregation/v0alpha1/types.go#L16) in the data plane aggregator. status has a [separate strategy](https://github.com/grafana/grafana/blob/eecd56659c4f3512b27cb6d741540fe67ae1b069/pkg/aggregator/registry/dataplaneservice/strategy.go#L91) and wraps the [parent storage](https://github.com/grafana/grafana/blob/eecd56659c4f3512b27cb6d741540fe67ae1b069/pkg/aggregator/registry/dataplaneservice/storage/storage.go#L108). it's then [registered as a subresource](https://github.com/grafana/grafana/blob/eecd56659c4f3512b27cb6d741540fe67ae1b069/pkg/aggregator/registry/dataplaneservice/rest/storage.go#L25) so that RBAC can limit write access to that field

`Groups` might also belong in `status` if it's updated in a similar way to `DashboardTitle`

Reply:
Want to make this property available in the API in main, so FE can do development.

So will proceed with this, but follow up separately on moving from the `spec` to `status` for the container of these properties.